### ISSUE TITLE
Checking the availability of repo 

### ISSUE THREAD

**Issue by AishaAgarwal** [upes-open/Lost-And-Found-Mobile-App#50]
Is this still repo and can I contribute to it still ??

**Comment by aakshitaa** [comment]
> Is this still repo and can I contribute to it still ??

Yeah sure u can contribute, just tell me which issue you wanna work on

**Comment by AishaAgarwal** [comment]
Cool ..will look up and let you know ASAP

**Comment by AishaAgarwal** [comment]
Also, Can you help me with the steps for running the project as I am not able to do so

**Comment by aakshitaa** [comment]
> Also, Can you help me with the steps for running the project as I am not able to do so

Yeah you can refer to some youtube videos for running react native apps and you need to have expo go if you want to run in on your phone

**Comment by AishaAgarwal** [comment]
Understood ..after doing so I would like to add that in the readme file

**Comment by AishaAgarwal** [comment]
Understood ..after doing so I would like to add that in the readme file

**Comment by AishaAgarwal** [comment]
I am not able to run the application using expo go ..as i am facing some path issue 
![image](https://github.com/upes-open/Lost-And-Found-Mobile-App/assets/107138192/c20e56f1-c70f-46fb-bc1f-d33e3b316132)
Please help me as needed

**Comment by aakshitaa** [comment]
After cloning the repo move to the main folder which contains the node packages, there run "npm install" and then once all the packages are installed then run "expo start"
Do this much and let me know if you face any errors.

**Comment by AishaAgarwal** [comment]
npm install only is giving this error 
![image](https://github.com/upes-open/Lost-And-Found-Mobile-App/assets/107138192/3c0ae935-8d2f-4762-aa3b-0aca1008e556)
